dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -103,7 +103,7 @@ def exit_survey(workshop, teacher, enrollment)
     mail content_type: content_type,
       from: from_hadi,
       subject: 'How was your Code.org workshop?',
-      to: email_address(@teacher.name, @teacher.email)
+      to: email_address(@enrollment.name || @teacher.name, @teacher.email)
```

Comment:
@tanyaparker @breville I was thinking more about this, and since we ask for and store the name separately in the workshop enrollment from the user account settings, we could ask for separate first and last name at least there, in enrollment. Either way we may have a discrepancy between that and the account, but that's an orthogonal issue.

Now if we make the change, what should we do with the existing enrollments that don't have separate first and last names? Should we attempt to upgrade them and just split the existing name based on whitespace, presuming "first last"?